docker/build.Dockerfile

```diff
@@ -87,12 +89,5 @@ RUN  CXX_COMPILER="g++" \
     && cmake --build . --target check \
     && cmake --build . --target install
 
-# Enable perfomance layers and test with `vkcube`.
-RUN export LD_LIBRARY_PATH=/performance-layers/build:$LD_LIBRARY_PATH \
-    && export VK_INSTANCE_LAYERS=VK_LAYER_STADIA_pipeline_compile_time \
-    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_runtime \
-    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_cache_sideload \
-    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_memory_usage \
-    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_frame_time \
-    && export VK_LAYER_PATH=/performance-layers/layer \
-    && xvfb-run vkcube --c 100 && echo Done!
+# Enable and test the performance layers.
+RUN . /performance-layers/docker/perf_layer_env.sh /performance-layers/build /performance-layers/layer
```

Comment:
1. Why do we need the leading `.`? I would think that
2. nit: missing newline at the end of the file
3. `perf_layer_env.sh` is not a very descriptive name. Consider something like `test_performance_layers.sh` or `check_performance_layers.sh`

Reply:
We either need to turn the script into an executable file or use `source`, `bash`, or `.`. I'll change it to the `bash` command.

Reply:
> We either need to turn the script into an executable file

This seems like the best option to me. It even already starts with a shebang suggesting it's executable